pkg/service/collector_test.go

```diff
@@ -66,13 +66,13 @@ func TestCollectorGRPCPortName(t *testing.T) {
 		{
 			"nil",
 			nil,
-			"grpc-http",
+			"grpc-model-proto",
```

Comment:
I would be more specific and call it `grpc-jaeger`  It as well aligns with OTEL port name. 

There is only a single Jaeger ingestion gprc protocol and I don't think there will be another one.

Reply:
I wonder if then `tls-grpc-jaeger` would work too. I would prefer that name over `https-jaeger` just to avoid future confusion. :x: 

![grafik](https://user-images.githubusercontent.com/10403402/197491786-5ecec041-6f4a-4c23-9bba-0417010699b6.png)

Reply:
Maybe we should ask review form the istio/maistra team on this.

Reply:
y, iam waiting. :)

Reply:
On a test cluster @iblancasa and I could confirm that replacing the `https` prefix with `tls` worked fine.